apps/src/netsim/NetSimMetronome.html.ejs

```diff
@@ -0,0 +1,85 @@
+<%
+  function moveCommand(x, y) {
+    return 'M' + x + ' ' + y;
+  }
+
+  function lineCommand(x, y) {
+    return 'L ' + x + ' ' + y;
+  }
+
+  function arcCommand(rX, rY, rot, largeArcFlag, sweepFlag, finalX, finalY) {
+    return 'A ' + rX + ' ' + rY + ', ' + rot + ', ' + largeArcFlag + ', ' + sweepFlag + ', ' + finalX + ' ' + finalY;
+  }
+
+  function closeShapeCommand() {
+    return 'Z';
+  }
+
+  /**
+   * Outer radius of progress border
+   * @type {number}
+   * @const
+   */
+  var OUTER_RADUS = 45;
+
+  /**
+   * Inner radius of progress border, and maximum radius of pulse circle
+   * @type {number}
+   * @const
+   */
+  var INNER_RADIUS = 30;
+
+  /**
+   * How long (in ms) it takes the pulse circle to shrink back to nothing after it appears
+   * @type {number}
+   * @const
+   */
+  var PULSE_FALLOFF_MS = 500;
+
+  /**
+   * Radius of the pulse circle, based on the pulse age.  Starts full size, then shrinks
+   * to r=0 over PULSE_FALLOFF_MS.
+   * @type {number}
+   */
+  var pulseCircleRadius = INNER_RADIUS * (1 - Math.min(pulseAge / PULSE_FALLOFF_MS, 1));
```

Comment:
Could extract second part of expression to a temporary variable to improve readability.
```
// Completion progress of pulse falloff, 0 at top of pulse, 1 when falloff is complete.
var pulseFalloffProgress = Math.min(pulseAge / PULSE_FALLOFF_MS, 1);
var pulseCircleRadius = INNER_RADIUS * (1 - pulseFalloffProgress);
```